homeassistant/components/lutron_caseta/config_flow.py

```diff
@@ -66,7 +65,7 @@ async def async_step_user(self, user_input=None):
 
     async def async_step_zeroconf(self, discovery_info):
         """Handle a flow initialized by zeroconf discovery."""
-        hostname = discovery_info[ATTR_HOSTNAME]
+        hostname = discovery_info["hostname"]
```

Comment:
I actually believe this is a bug. And it should be `discovery_info[CONF_HOST]` as in other similar places.

Reply:
Or not. Test sets both fields.